Example/Podfile.lock

```diff
@@ -14,15 +14,15 @@ DEPENDENCIES:
 
 EXTERNAL SOURCES:
   loudspeaker:
-    :path: ../
+    :path: "../"
 
 SPEC CHECKSUMS:
-  Expecta: 3b6bd90a64b9a1dcb0b70aa0e10a7f8f631667d5
-  loudspeaker: 33a267e6dc5da9c7cf98e65b6c4e17a3d39acc10
-  Masonry: 281802d04d787ea2973179ee8bcb50500579ede2
-  OCMock: 35ae71d6a8fcc1b59434d561d1520b9dd4f15765
-  Specta: 3e1bd89c3517421982dc4d1c992503e48bd5fe66
+  Expecta: e1c022fcd33910b6be89c291d2775b3fe27a89fe
+  loudspeaker: 55f868639bb17d57c38fea1752848ef20e762e88
+  Masonry: 678fab65091a9290e40e2832a55e7ab731aad201
+  OCMock: f3f61e6eaa16038c30caa5798c5e49d3307b6f22
+  Specta: ac94d110b865115fe60ff2c6d7281053c6f8e8a2
 
 PODFILE CHECKSUM: f09561418a350e655ac9af3b4c31e8a50323005f
 
-COCOAPODS: 1.3.1
+COCOAPODS: 1.1.1
```

Comment:
This lockfile was still generated with an older version of CocoaPods. Try `bundle exec pod install` to fix.